int64 human_action
int64 human_expression
int64 object_size
---
float64 p_happy
float64 p_sad
float64 p_neutral


